moped-editor/src/queries/components.js

```diff
@@ -146,9 +150,21 @@ export const UPDATE_COMPONENT_FEATURES = gql`
     insert_feature_signals(objects: $signals) {
       affected_rows
     }
+    insert_feature_drawn_lines(objects: $drawnLines) {
+      affected_rows
+    }
+    insert_feature_drawn_points(objects: $drawnPoints) {
+      affected_rows
+    }
     update_features_many(updates: $updates) {
       affected_rows
     }
+    update_feature_drawn_lines_many(updates: $drawnLinesDragUpdates) {
```

Comment:
I wanted to be very explicit that these updates come from dragging a line or point on the map to update its coordinates. [This issue](https://github.com/cityofaustin/atd-data-tech/issues/10716) will potentially open up other kinds of updates, and we can extend this if needed then.